tests/foreman/ui/test_rhai.py

```diff
@@ -0,0 +1,136 @@
+"""Tests for Red Hat Access Insights"""
+
+
+from nailgun import entities
+from fauxfactory import gen_string
+from robottelo.common import conf
+from robottelo.common import manifests
+from robottelo.ui.navigator import Navigator
+from robottelo.ui.session import Session
+from robottelo.test import UITestCase
+from robottelo.vm import VirtualMachine
+
+
+class RHAITestCase(UITestCase):
+    def test_rhai_client_setup(self):
+        """@Test: Check for client registration to
+        redhat-access-insights service.
+
+        @Feature: RHEL client registration to rhai
+
+        @Assert: Registered client should appear in the Systems
+        sub-menu of Red Hat Access Insights
+        """
+        activation_key_name = gen_string('alpha')
+
+        # Create a new organization with prefix 'insights'
+        org = entities.Organization(
+            name='insights_{0}'.format(gen_string("alpha", 6))
+        ).create()
+
+        # Create new lifecycle environment
+        lifecycle_env = entities.LifecycleEnvironment(
+            name='rhai_env_{0}'.format(gen_string("alpha", 6)),
+            organization=org,
+        ).create()
+
+        # Upload manifest
+        manifest_path = manifests.clone()
+        org.upload_manifest(path=manifest_path)
+
+        # Create a custom product for RHAI
+        custom_product = entities.Product(
+            name='rhai_product_{0}'.format(gen_string("alpha", 6)),
+            organization=org,
+        ).create()
+
+        # Add insights repo to the custom product
+        custom_repo = entities.Repository(
+            content_type=u'yum',
+            name=u'insights_repo_{0}'.format(gen_string('alpha', 6)),
+            product=custom_product,
+            url=u'http://access.redhat.com/insights/repo/6/',
+        ).create()
+
+        # Sync custom repository
+        custom_repo.sync()
+
+        # Create content view
+        content_view = entities.ContentView(
+            name='rhai_cv_{0}'.format(gen_string("alpha", 6)),
```

Comment:
Nitpick: please use single quotes as shown in https://github.com/SatelliteQE/robottelo/pull/2567#issuecomment-121968947